js/components/moter/motebehov/MotebehovKvitteringUtvidbar.js

```diff
@@ -12,33 +13,31 @@ const tekster = {
   },
 };
 
-const getHarBehovKvittering = (harBehovSvar, harBehovSporsmal) => {
+export const getHarBehovKvittering = (harBehovSvar, harBehovSporsmal) => {
```

Comment:
Synes at det er bedre hvis vi forholder oss til enten "behov" eller "motebehov" som begrep og ikke bruker begge samtidig. Dette er jo gammel fil, men det kan være nyttig å oppfriske sånne ting slik at de ikke blir glemt i ny kode og :)

Reply:
Enig, vil helst ikke røre for mye av de eldre filene. Vi må definitivt ha det i bakhodet når vi skriver om :)

Reply:
> Det er flere lint-feil i filer du ikke har berørt ipå denne branchen som kan fikses når de mindre branchene merges til `dm-innkalelse` 👍🏻

Yepp. Tenker å fikse det i en annen branch som er branchet ut fra denne.